### DISCUSSION TITLE
Improving chat discoverability and accessibility

### DISCUSSION THREAD

**Discussion by axelsimon** [openbao/openbao#208] [Ideas]
As a long time Element / Matrix user, i'd like to see the openbao chat a bit easier to find and join.

Currently, the README links to the Element web client at https://chat.lfx.linuxfoundation.org which expects folks to log in with a LF account. Is this by design?

I could understand if it is, as this provides some level of control over who can chat and helps with moderation and spam issues. But on the other hand as an open project, using a decentralised comms protocol, i was hoping to be able to find the openbao rooms using my existing Matrix account in Element.

The issue is that currently, these rooms neither have a public alias on, say, matrix.org (such as `#openbao-general:matrix.org`) nor does the `chat.lfx.linuxfoundation.org` server allow rooms to be browsed or searched by external users.

Any thoughts?

**Comment by joewxboy** [comment]
@axelsimon Thank you for that great description and summary of how the Matrix chat server is currently configured to function.  The server is provided by the Linux Foundation's services team (LFX).  Let me ask them if they could explain why it is configured this way (to require SSO, to have the rooms not be discoverable outside the organization).

**Comment by cipherboy** [comment]
We've recently added more links to our org's [README](https://github.com/openbao#contact) and [website](https://openbao.org/docs/contributing/#chat-server) to help improve discoverability. However, while I've also followed up with the LF around their matrix server versions, I've also not gotten any responses. So, I'm going to go ahead and close this anyways.
